src/command-parser/expand-npm-wildcard.js

```diff
@@ -28,7 +28,7 @@ module.exports = class ExpandNpmWildcard {
             .filter(script => wildcardRegex.test(script))
             .map(script => Object.assign({}, commandInfo, {
```

Comment:
I rolled this line back to what it is on `master`, as some other information will exist on `commandInfo`